src/core/icons/icon-gui-sliders.svg

```diff
@@ -0,0 +1,11 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Are these from [Heroicons](https://heroicons.com/outline) or exported from Figma? 

Asking as my last icon additions were updated with their Heroicons equivalent. In this case it would be, `adjustments-horizontal`. 

Either way would be useful to clear up where the source of the icons should come from :)

Reply:
You're right, they're heroicons. It seems like the time has come where the hero icons have started turning up in the designs - I guess we can either continue to have piecemeal PRs adding them in a couple at a time, or just rip the plaster off and introduce the (24px outline) hero icons into the ably-ui sprite sheet with their original names, replacing any identically named legacy ones.

I'll look into it and likely change this to a minor version bump